src/ape/managers/converters.py

```diff
@@ -38,6 +45,28 @@ def convert(self, value: str) -> bytes:
         return HexBytes(value)
 
 
+class HexIntConverter(ConverterAPI):
+    """
+    Convert hex values to integers.
+    """
+
+    def is_convertible(self, value: Any) -> bool:
+        return (
+            isinstance(value, int)
+            or (isinstance(value, str) and is_hex(value))
+            or isinstance(value, bytes)
+        )
+
+    def convert(self, value: Any) -> int:
+        if isinstance(value, str) and is_hex(value):
+            return to_int(HexBytes(value))
```

Comment:
Can probably reduce it to just this, since it should be guaranteed to be a hex string or bytes coming in

Reply:
https://github.com/ApeWorX/ape/pull/1248